package.json

```diff
@@ -1,7 +1,9 @@
 {
   "name": "@fieldguide/pipeline",
   "description": "A toolkit to easily build synchronous process pipelines in TypeScript/JavaScript",
-  "version": "0.1.0",
+  "version": "0.0.1",
```

Comment:
Minor, but if you run `npm i` again, this version change will update `package-lock.json`.

Reply:
Great catch. I've updated it in 1b12284.